spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -995,6 +995,31 @@ def test_fallback_completions(fallback_codeeditor, qtbot):
     code_editor.toggle_code_snippets(True)
 
 
+@pytest.mark.slow
+@pytest.mark.first
+@flaky(max_runs=5)
+def test_text_snippet_completions(snippets_codeeditor, qtbot):
+    code_editor, _ = snippets_codeeditor
+    completion = code_editor.completion_widget
+
+    code_editor.toggle_automatic_completions(False)
+    code_editor.toggle_code_snippets(False)
+
+    # Set cursor to start
+    code_editor.go_to_line(1)
+
+    with qtbot.waitSignal(completion.sig_show_completions,
+                          timeout=10000) as sig:
+        qtbot.keyClicks(code_editor, 'f')
+        qtbot.keyPress(code_editor, Qt.Key_Tab, delay=300)
+
+    # Assert all retrieved words start with 'f'
+    assert all({x['label'].startswith('f') for x in sig.args[0]})
```

Comment:
Could we also test at this point the specific names of some `snippets` (such as `for` or `from`)?